app/car.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+
+class Car:
+    cars = []
+
+    def __init__(
+            self,
+            brand: str,
+            fuel_consumption: float
+    ) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    @classmethod
+    def add_to_car(cls) -> None:
```

Comment:
rename to load_from_json it is more logical, and better to pass file_path as parameter